topi/python/topi/x86/conv2d.py

```diff
@@ -310,6 +292,11 @@ def _declaration_conv_impl(cfg, data, kernel, strides, padding, layout, out_dtyp
     else:
         data_pad = data
 
+    if cfg.is_fallback:
+        wkl = Workload(data.dtype, out_dtype, in_height, in_width, in_channel, num_filter,
+                       kernel_height, kernel_width, HPAD, WPAD, HSTR, WSTR)
+        cfg = _get_default_config(wkl)
```

Comment:
instead of returning a new config, we can directly modify the values in fallback schedule. e.g. cfg['tile_ic'] = 12.
The same modified config will also be passed to topi_schedule, so we can to handle fallback only in compute.

Reply:
I add `__setitem__` to `FallbackConfigEntity`, if it looks good.